File: app/src/main/java/me/bilalhaider/moviedatabase/data/model/SearchResponse.java
package me.bilalhaider.moviedatabase.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class SearchResponse{
	@SerializedName("Response")
	private String response;
	@SerializedName("totalResults")
	private String totalResults;
	@SerializedName("Search")
	private List<SearchItemResponse> search;

	public void setResponse(String response){
		this.response = response;
	}

	public String getResponse(){
		return response;
	}

	public void setTotalResults(String totalResults){
		this.totalResults = totalResults;
	}

	public String getTotalResults(){
		return totalResults;
	}

	public void setSearch(List<SearchItemResponse> search){
		this.search = search;
	}

	public List<SearchItemResponse> getSearch(){
		return search;
	}
}

File: network/mobileapi-client/src/commonMain/kotlin/me/bilalhaider/moviedatabase/network/mobileapi/client/response/SearchResponse.kt
package me.bilalhaider.moviedatabase.network.mobileapi.client.response

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import me.bilalhaider.moviedatabase.network.model.SearchResult

/**
 * Created by Bilal Haider on 17/03/2022
 */
@Serializable
data class SearchResponse(
    @SerialName("Search") val data: List<SearchResult> = emptyList(),
    val totalResults: String = ""
)